src/eventbrite-provider.ts

```diff
@@ -9,22 +11,36 @@ function EventbriteProvider(this: any, options: any) {
 
   let API_KEY = ''
 
+  let eventbriteSDK: Sdk
 
-  // seneca.message('role:entity,cmd:load')
-
+  seneca
+    .message('role:entity,cmd:load,base:eventbrite,name:event', loadEvent)
 
 
   seneca.prepare(async function(this: any) {
     let out = await this.post('sys:provider,get:key,provider:eventbrite,key:api')
     if (out.ok) {
       API_KEY = out.value
+      eventbriteSDK = Eventbrite({token: API_KEY})
     }
     else {
       this.fail('api-key-missing')
     }
   })
 
+  async function loadEvent(this: any, msg: any) {
+    const q: any = msg.q
+    const eventID: string = q.id
+
+    const event: any = await eventbriteSDK.request(`/events/${eventID}`)
+
+    if(event.id) {
```

Comment:
Whereas with github, we were using the `owner/repo` string as the ID, this does not apply for eventbrite. In this case I don't think we need to manipulate the ID

Reply:
In fact the events identifiers are integers and that statement are not useful here. Thank you for the feedback !